t2.cpp

```diff
@@ -0,0 +1,6 @@
+sdjfsjsdjfsd;
+fsdfsdfsf;
+sdfdsfdsf;
```

Comment:
variable 'sdfdsfdsf' can be made static or moved into an anonymous namespace to enforce internal linkage [misc-use-internal-linkage,-warnings-as-errors]